rc_int

```diff
@@ -0,0 +1,19 @@
+export APACHE_BASE_PATH=main
+export API_URL=//mf-chsdi3.int.bgdi.ch
+export DBHOST=pg.bgdi.ch
```

Comment:
At the moment `buildout_int.cfg` extends `buildout.cfg` and we have `pg.bgdi.ch` see https://github.com/geoadmin/mf-chsdi3/blob/master/buildout.cfg#L105
This may be an error in current buidlout system though. @ltclm what instance are we suppose to point to in `int`?